helm/default-apps-aws/values.yaml

```diff
@@ -7,6 +7,12 @@ userConfig:
       values: |
         ipam:
           mode: kubernetes
+        k8sServiceHost: api.{{ .Values.baseDomain }}
+        k8sServicePort: 6443
```

Comment:
Isn't it 443 if it's using the LB?

Reply:
It's not https://github.com/giantswarm/giantswarm/issues/22518. I actually tested it with 6443.